src/components/include/transport_manager/transport_manager.h

```diff
@@ -56,11 +53,16 @@ class TransportManager {
    **/
   virtual ~TransportManager() {}
 
+  DEPRECATED
+  virtual int Init(resumption::LastState&) {
+    return 1;
```

Comment:
@YarikMamykin what is this?

Reply:
@AKalinich-Luxoft this was provided with 2dd6f73cf75d7c17d86e0313dde501d3766e61c2. It is hard for me to answer why this `DEPRECATED Init` always returned 1)))